packages/font-icons-mdl2/src/index.ts

```diff
@@ -23,19 +23,28 @@ import { registerIconAliases } from './iconAliases';
 import { getWindow } from '@fluentui/utilities';
 const DEFAULT_BASE_URL = 'https://spoppe-b.azureedge.net/files/fabric-cdn-prod_20210407.001/assets/icons/';
 
+/*
+ * The Window variable has the iconBaseUrl prop in order to allow for users to redirect icon font downloads to a new URL.
+ * The config can be burned on the page to ensure there are no race conditions which might load resources on script load.
+ */
 declare global {
   // eslint-disable-next-line @typescript-eslint/naming-convention
   interface Window {
     FabricConfig?: {
+      /**
+       * @deprecated - Use iconBaseUrl instead.
+       */
       fontBaseUrl?: string;
+      iconBaseUrl?: string;
```

Comment:
Note: I've removed deprecated flag in the suggestion below. We don't need to deprecate the
`fontBaseUrl` setting; fonts can still be loaded using it.

```suggestion
  interface Window {
  /**
   * The FabricConfig options can be burned on the page prior to script load to provide
   * alternative defaults at script load time.
   */
    FabricConfig?: {
      /**
       * Controls the base url of the font files. This is useful for scenarios where the fonts
       * are stored on a private CDN other than the default SharePoint CDN.
       */
      fontBaseUrl?: string;

      /**
       * Controls the base url of the icon font files. This is useful for scenarios where the icons
       * are stored on a private CDN other than the default SharePoint CDN. Note that in prior
       * iterations, `fontBaseUrl` was used to control both font and icon base urls. While you can
       * still use `fontBaseUrl` to provide a single base url for both, the `iconBaseUrl` will be
       * used first if available.
       */
      iconBaseUrl?: string;
```